app/models/event/participation_contact_datas/manager.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2023, CEVI Schweiz, Pfadibewegung Schweiz,
+#  Jungwacht Blauring Schweiz, Pro Natura, Stiftung für junge Auslandschweizer.
+#  This file is part of hitobito_youth and
+#  licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_youth.
+
+class Event::ParticipationContactDatas::Manager < Event::ParticipationContactData
+  self.mandatory_contact_attrs = [:first_name, :last_name, :email]
```

Comment:
Warum musst du diese nochmals setzen? Das ist doch ein class_attribute, das müsste doch vererbt werden? Und in `Event::ParticipationContactData` ist der Wert aktuell `[:email, :first_name, :last_name]`